### PR TITLE
Navigator support for SDC

### PR DESCRIPTION
**Pull request by kirtiv1**

### CODE REVIEW

File: start.py [hunk 1]
```diff
@@ -169,6 +185,19 @@ def main(args):
...
+                navigator_stage_lib = _get_navigator_stage_lib_name(args.cm_version,
+                                                                    args.additional_stage_libs_version)
+                stage_lib_name = '{}/{}/additional-datacollector-libs:{}'
+                navigator_stage_lib_image = stage_lib_name.format(args.registry,
+                                                                  args.namespace or DEFAULT_NAMESPACE,
+                                                                  navigator_stage_lib)
+                logger.info('Adding stage lib image for Navigator %s to primary node ... ', navigator_stage_lib_image)
+                primary_node.volumes.append(navigator_stage_lib_image)
+            except:
+                raise Exception('No Navigator stage library found for CM version %s.', args.cm_version)
```
Comment (dimaspivak) [inline start.py]: You can't use Exception like this (I think you're thinking of the `logging` modules). Gotta do `Exception('...'.format())`

File: start.py [hunk 2]
```diff
@@ -169,6 +185,19 @@ def main(args):
...
 
+        if args.navigator:
+            try:
+                navigator_stage_lib = _get_navigator_stage_lib_name(args.cm_version,
+                                                                    args.additional_stage_libs_version)
+                stage_lib_name = '{}/{}/additional-datacollector-libs:{}'
+                navigator_stage_lib_image = stage_lib_name.format(args.registry,
+                                                                  args.namespace or DEFAULT_NAMESPACE,
+                                                                  navigator_stage_lib)
+                logger.info('Adding stage lib image for Navigator %s to primary node ... ', navigator_stage_lib_image)
```
Comment (dimaspivak) [inline start.py]: I'd make this `debug`

File: start.py [hunk 3]
```diff
@@ -1129,10 +1164,19 @@ def _setup_ssl_encryption_authentication(cluster, service):
     cluster.primary_node.execute(' && '.join(ssl_authentication_commands + ssl_commands))
 
 
+def _get_navigator_stage_lib_name(cm_version, additional_stage_libs_version=None):
+    # Returns version as a tuple (e.g. ('5', '15', '0')).
```
Comment (dimaspivak) [inline start.py]: Can probably remove this comment (obvious from the code what the next line does :-p)

**Review by dimaspivak — CHANGES_REQUESTED** [review]
Just a handful of tiny nits. Good work!

File: start.py (hunk 2 above)
Comment (kirtiv1) [reply start.py]: Done.

File: start.py (hunk 1 above)
Comment (kirtiv1) [reply start.py]: Done.

File: start.py (hunk 3 above)
Comment (kirtiv1) [reply start.py]: Done.

**Review by dimaspivak — APPROVED** [review]

**Pull request merged**